src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -205,80 +205,78 @@ ValidatorList::applyList (
 
     Json::Value list;
     PublicKey pubKey;
-    auto const result = verify (list, pubKey, manifest, blob, signature);
+    auto const result = verify(list, pubKey, manifest, blob, signature);
     if (result != ListDisposition::accepted)
         return result;
 
     // Update publisher's list
     Json::Value const& newList = list["validators"];
     publisherLists_[pubKey].available = true;
-    publisherLists_[pubKey].sequence = list["sequence"].asUInt ();
+    publisherLists_[pubKey].sequence = list["sequence"].asUInt();
     publisherLists_[pubKey].expiration = TimeKeeper::time_point{
         TimeKeeper::duration{list["expiration"].asUInt()}};
-    std::vector<PublicKey>& publisherList = publisherLists_[pubKey].list;
+    auto& publisherList = publisherLists_[pubKey].list;
 
-    std::vector<PublicKey> oldList = publisherList;
-    publisherList.clear ();
-    publisherList.reserve (newList.size ());
+    auto oldList = publisherList;
```

Comment:
What about 
```
auto oldList = std::move(publisherList);
publisherList.reserve(newList.size());
```
Old code was making a copy + reallocating, this would save us from the copy.